drivers/google/google.go

```diff
@@ -260,20 +271,21 @@ func (d *Driver) Remove() error {
 	return c.deleteDisk()
 }
 
-// Restart deletes and recreates the GCE instance, keeping the disk.
 func (d *Driver) Restart() error {
+	return nil
+}
+
+// Kill deletes the GCE instance, but keeps the disk.
```

Comment:
I'm not sure I understand what's the spirit of it.

Before the patch, Kill() would delegate to Stop(). And Stop() would delete the instance keeping the disk. This comes from a time where GCE instances couldn't be stopped.

I thought Stop() should keep the instance but keep it. And Kill() should delete the instance but keep the disk so that the instance could be recreated.

WDYT?